test/adagrams.test.js

```diff
@@ -120,7 +120,7 @@ describe("Adagrams", () => {
     });
 
     it("returns a score of 0 if given an empty input", () => {
-      throw "Complete test";
+      expectScores({});
```

Comment:
@cornetto9, please complete this assertion. You can add it here as a reply to my comment or push up your change.

You provide an empty object, but we want to test that when the input is an empty string "" then **we get back a score of 0.** 

The empty object doesn't test that an empty string input returns a score of 0 as written.